contrib/linux-kernel/linux_zstd.h

```diff
@@ -136,10 +136,20 @@ typedef ZSTD_parameters zstd_parameters;
 zstd_parameters zstd_get_params(int level,
 	unsigned long long estimated_src_size);
 
-/* ======   Single-pass Compression   ====== */
-
 typedef ZSTD_CCtx zstd_cctx;
 
+/**
+ * zstd_cctx_set_param() - sets a compression parameter
+ * @cctx:         The context. Must have been initialized with zstd_init_cctx().
+ * @param:        The parameter to set.
+ * @value:        The value to set the parameter to.
+ *
+ * Return:        Zero or an error, which can be checked using zstd_is_error().
+ */
+size_t zstd_cctx_set_param(zstd_cctx *cctx, ZSTD_cParameter param, int value);
```

Comment:
minor: We could also add a typedef for `zstd_cparameter` to give it a kernel-style name.

Reply:
I could go either way. I don't think we absolutely need to, given that we don't have many users of this API.